web/server/vue-cli/package-lock.json

```diff
@@ -2931,6 +2931,79 @@
         "minimist": "^1.2.0"
       }
     },
+    "@eslint/eslintrc": {
```

Comment:
This file should contain changes related only to the codechecker_api package (maybe your npm version is too new. I am using 6.14.4 version.)